components/edit-collective/sections/Members.js

```diff
@@ -85,7 +85,7 @@ class Members extends React.Component {
       },
       removeConfirm: {
         id: 'members.remove.confirm',
-        defaultMessage: `Do you really want to remove {name} @{slug} {hasEmail, select, 1 {({email})} other {}} from the core contributors of the collective?`,
+        defaultMessage: `Do you really want to remove {name} @{slug} {hasEmail, select, 1 {({email})} other {}} from the team?`,
```

Comment:
We can just say "Do you really want to remove [name]" and delete the phrase "from the...." here. Better to simplify where possible.